frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,93 @@
+Feature: Image Vulnerability in Project
+    As a user, I would like to see Image Vulnerability in Project
+
+Background:
+    Given user is in Developer perspecitve
+    And user in project namespace "aut-image-vulnerabilty"
+    And quay container security operator installed
+
+@regression
+Scenario: Vulnerability in project overview    
+   Given user is in topology page
```

Comment:
```suggestion
   Given user is at topology page
```

Reply:
What difference it would create

Reply:
We already have a step definition written for that statement